README.md

```diff
@@ -1,2 +1,23 @@
-# hw01
-Repository for homework 01
+# **This is the Repository of Charles Braeden McNulty**
+My name is Braeden McNulty, a Political Science MAPSS student from [Pikeveille, KY.](http://pikevilleky.gov/)
+After graduating from Pikeville High School, I atteneded [Centre College](https://www.centre.edu/) in Danville, KY where I earned a BA in _Political Science_ and _History_. At Centre, I was involved numverous organizations that allowed me to share my love of my school with others. These include: 
+
+1. Student Government
+2. First Year Orientation
```

Comment:
You don't need to assign the numbers yourself. For example,

```
1. Student Government
1. First Year Orientation
1. Admissions Tour Guide
```

renders as

1. Student Government
1. First Year Orientation
1. Admissions Tour Guide